nvtabular/ops/add_metadata.py

```diff
@@ -36,6 +34,42 @@ def transform(self, col_selector: ColumnSelector, df: DataFrameType) -> DataFram
     def output_tags(self):
         return self.tags
 
+
+class AddMetadata(AddTags):
```

Comment:
Seems like the goal with `AddTags` is to provide a more intuitive operator name, so it seems like the simplest way to do that is probably to make `AddTags` a subclass of `AddMetadata` where the `init` only accepts tags and `output_tags` returns them, and then to create a second sub-class called `AddProperties` that does the same thing but via `output_properties`

Reply:
(That way we can maintain the ability to add tags and properties simultaneously with `AddMetadata`, and still provide operators with more intuitive names.)